src/test/java/org/generationcp/breeding/manager/listmanager/util/GermplasmListExporterTest.java

```diff
@@ -659,4 +683,46 @@ public static void cleanUp() {
 
 	}
 
+	@Test
+	public void testGetVisibleColumnMapWithMGID() {
+		Map<String, Boolean> visibleColumnsMap;
+		int visibleColumnCount;
+		GermplasmListExporterTest.listDataTable = GermplasmListExporterTest.generateTestTable(new ArrayList<>(Arrays.asList(ColumnLabels.MGID.getName())));
+
+		visibleColumnsMap = this.germplasmListExporter.getVisibleColumnMap(GermplasmListExporterTest.listDataTable);
+		visibleColumnCount = this.getNoOfVisibleColumns(visibleColumnsMap);
+		Assert.assertEquals(10, visibleColumnCount);
+		Assert.assertTrue("Expected to have exactly 10 visible columns.", visibleColumnCount == 10);
```

Comment:
Same here. `assertTrue` seems redundant to line before

Reply:
In addition to counting the visible columns, an assertion that FGID/MGID column is not included in the `visibleColumnsMap` will make this test stronger